#! /bin/sh /usr/share/dpatch/dpatch-run
## DP: Produces shorted doxygen filenames for ACE. Taken from SVN.
@DPATCH@
--- ACE_wrappers.orig/etc/ace.doxygen
+++ ACE_wrappers/etc/ace.doxygen
@@ -34,6 +34,7 @@
 DISTRIBUTE_GROUP_DOC  = YES
 TAB_SIZE              = 8
 ENABLED_SECTIONS      =
+SHORT_NAMES           = YES
 
 #---------------------------------------------------------------------------
 # configuration options related to warning and progress messages
